data.md

```diff
@@ -83,7 +87,9 @@ WebAssembly has four basic types, for 32 and 64 bit integers and floats.
 ```k
     syntax IValType ::= "i32" | "i64"
     syntax FValType ::= "f32" | "f64"
-    syntax  ValType ::= IValType | FValType
+    syntax AValType ::= IValType | FValType
+    syntax NValType ::= "{" Identifier AValType "}"
+    syntax ValType  ::= AValType | NValType
  // ---------------------------------------
```

Comment:
Can you make sure the documentation makes clear what `A` and `N` stand for here? I don't get the naming.